transformation-scripts/cmip6-kerchunk/publish-cmip6-kerchunk-stac.ipynb

```diff
@@ -0,0 +1,309 @@
+   "cell_type": "markdown",
+   "id": "e256d2e4-6ff0-461d-a847-794aae86e444",
+   "metadata": {},
+   "source": [
+    "## Step 2: Open the dataset with `xarray`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "0d11fbe9-f562-481c-8648-f43b17896665",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/tmp/ipykernel_5419/732403854.py:16: UserWarning: Variable(s) referenced in cell_measures not in variables: ['areacella']\n",
+      "  ds = xr.open_dataset(src_path, **xr_open_args)\n"
+     ]
+    }
+   ],
+   "source": [
+    "dataset_url = 's3://veda-data-store-staging/cmip6-GISS-E2-1-G-tas-kerchunk/combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk.json'\n",
+    "\n",
+    "xr_open_args = {\n",
+    "    \"engine\": \"zarr\",\n",
+    "    \"decode_coords\": \"all\",\n",
+    "    \"consolidated\": False\n",
+    "}\n",
+    "\n",
+    "fs = fsspec.filesystem(\n",
+    "    \"reference\",\n",
+    "    fo=dataset_url,\n",
+    "    remote_options={\"anon\": True},\n",
+    ")\n",
+    "src_path = fs.get_mapper(\"\")\n",
+    "\n",
+    "ds = xr.open_dataset(src_path, **xr_open_args)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "4449a055-779e-4024-a871-ddf107cb27d2",
+   "metadata": {
+    "tags": []
+   },
+   "source": [
+    "## Step 3: Generate STAC metadata\n",
+    "\n",
+    "The spatial extent is taken from the xarray metadata. The temporal extent will be added by the [`xstac`](https://github.com/stac-utils/xstac) library."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "47cba6c5-2d6c-4829-8b49-ada67e6b5522",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "spatial_extent_values = [ds.lon[0].values, ds.lat[0].values, ds.lon[-1].values, ds.lat[-1].values]\n",
+    "spatial_extent = list(map(int, spatial_extent_values))\n",
+    "_id = 'combined_CMIP6_daily_GISS-E2-1-G_tas_kerchunk_TEST'\n",
+    "zarr_asset = pystac.Asset(\n",
+    "    title='zarr',\n",
+    "    href=dataset_url,\n",
+    "    media_type='application/vnd+zarr',\n",
+    "    roles=['data'],\n",
+    ")\n",
+    "extent = pystac.Extent(\n",
+    "    spatial=pystac.SpatialExtent(bboxes=[spatial_extent]),\n",
+    "    temporal=pystac.TemporalExtent([[None, None]])\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3351996d-f124-410d-a6a2-55abcdb2aecb",
+   "metadata": {},
+   "source": [
+    "Add the VEDA provider."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "aaef1ad4-d796-48d5-80da-f3a0fced5f0f",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "providers = [\n",
+    "    pystac.Provider(\n",
+    "        name=\"VEDA\",\n",
+    "        roles=[pystac.ProviderRole.PRODUCER, pystac.ProviderRole.PROCESSOR, pystac.ProviderRole.HOST],\n",
+    "        url=\"https://github.com/nasa-impact/veda-data-pipelines\",\n",
```

Comment:
restating the comment from https://github.com/NASA-IMPACT/veda-config/pull/323#issuecomment-1750758191 about `veda-data-pipelines`

Reply:
Looking at https://github.com/radiantearth/stac-spec/blob/master/collection-spec/collection-spec.md#provider-object I think this should probably be the VEDA dataset page or the VEDA docs page

Reply:
I'm going to link to VEDA docs dataset ingestion pages for now, anticipating that the veda-docs will include these notebooks soon.

Reply:
🤔 actually since it says:
>Homepage on which the provider describes the dataset and publishes contact information.
I am linking to the dashboard data catalog, which will include this dataset soon.